romana/kubernetes/conductor.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2016 Pani Networks
```

Comment:
My point is that there are two things:

1. CLI stuff
2. Adapter to 3-party env stuff (like k8s, OS)

and some things are kind of both.

And so -?